### DISCUSSION TITLE
Few Questions

### DISCUSSION THREAD

**Discussion by BobtheBob123456** [Kas-tle/java2bedrock.sh#122] [Converter Help] [answered]
Hey,
First of, once the resource pack is converted, can it be used for multi-player servers?

Secondly, if it can be used on multi-player, where do I put the custom mapping things.

Thanks,

**Comment by typicalsmc** [comment, marked as answer]
> Hey,
> 
> First of, once the resource pack is converted, can it be used for multi-player servers?
> 
> 
> 
> Secondly, if it can be used on multi-player, where do I put the custom mapping things.
> 
> 
> 
> 
> 
> Thanks,

yes you can use it in server, put it into the packs folder if you are using geyser
you have to use the feature/extensions branch of Geyser to use the mappings

**Reply by typicalsmc** [reply]
you can download the build of it here https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/feature%252Fextensions/58/

**Reply by BobtheBob123456** [reply]
Wait, so are the mappings required, or can you ignore them. Also, you you need to put the behavior, resource pack, and addon in the packs folder, or just the behavior and resource pack.

**Reply by typicalsmc** [reply]
just the resourcepack
yes, mappings are required

**Reply by BobtheBob123456** [reply]
Ok, so I replace the geyser I already have, with this, right? And where do I put the mappings?

**Reply by typicalsmc** [reply]
yes
in custom_mappings folder iirc
just read this https://github.com/GeyserCustomModelData/HowTo

**Reply by BobtheBob123456** [reply]
So if I got this right.

1. Install Geyser extension
2. Use the converter to convert the pack
3. use the addon/behavior pack to test the pack
4. Put the resource pack in the packs folder of geyser
5. Put the mapping in the mapping folder of geyser
6. Join the server an enjoy
